strats.md

```diff
@@ -950,7 +950,7 @@ A `comeInWithTemporaryBlue` entrance condition indicates that Samus must come in
 
 A `comeInBlueSpinning` entrance condition indicates that Samus must come in with a spin jump through the doorway, possibly while having blue speed, applicable to horizontal transitions. It has the following properties:
 
-  - _minTiles_: The minimum effective runway length required to obtain sufficient speed before the jump.
+  - _minTiles_: The minimum effective runway length required to obtain sufficient speed before the jump. This can be omitted if the strat does not require coming any with any particular amount of momentum.
```

Comment:
```suggestion
  - _minTiles_: The minimum effective runway length required to obtain sufficient speed before the jump. This can be omitted if the strat does not require coming in with any particular amount of momentum.
```